app/views/user_mailer/invitation.text.erb

```diff
@@ -1,18 +1,18 @@
 <% content_for :message do %>
-Hello from the <%= "#{ArchiveConfig.APP_NAME}" %>!
+<%= t "user_mailer.invitation.greeting", app_name: ArchiveConfig.APP_NAME %>
 
 <% if !@user_name.blank? %>
-<%= @user_name %> has invited you to join our beta!
+<%= t "user_mailer.invitation.has_invited", user_name: @user_name %>
 <% else %>
-You've been invited to join our beta!
+<%= t "user_mailer.invitation.been_invited_html" %>
 <% end %>
 
-The Archive of Our Own (AO3) is a free, noncommercial archive built by and for fans. We are multifannish and welcome all kinds of fanworks, including fanfiction, fanart, fanvids, and podfic. We run on the open-source otwarchive software, and our servers are owned by the nonprofit Organization for Transformative Works, which works to protect fan rights and preserve fanworks.
+<%= t "user_mailer.invitation.part1" %>
 
-Also, we have a lot of spiffy features for finding the fanworks you want, and for posting and sharing your work with others. There are lots more coming soon! Your feedback now during our beta can help us build an even better archive. \o/
+<%= t "user_mailer.invitation.part2" %>
 
-If you'd like to join us, please sign up at the following address:
+<%= t "user_mailer.invitation.part3_html" %>
```

Comment:
This should probably also be combined with the URL to allow Translation to control the position of the URL in the sentence.